assets/js/googlesitekit/datastore/user/user-input-settings.js

```diff
@@ -65,30 +75,50 @@ const baseInitialState = {
 
 const baseActions = {
 	/**
-	 * Sets user input settings.
+	 * Gets cached user input settings and save them to the data store.
 	 *
-	 * @since 1.19.0
+	 * @since n.e.x.t
 	 *
-	 * @param {Object} values User input settings.
-	 * @return {Object} Redux-style action.
+	 * @return {Object} Cached user input answer values.
 	 */
-	setUserInputSettings( values ) {
-		return {
-			type: SET_USER_INPUT_SETTINGS,
-			payload: { values },
+	*fetchCachedUserInputSettings() {
```

Comment:
This name is a bit off for two reasons:
* The `fetch*` prefix should only be used for the internal fetch actions related to a public API-based selector or action (typically using `createFetchStore`). The action here isn't that though.
* The function sets user input settings based on what is cached, which is not clear from the name (fetch implies it's getting/fetching something, but it also sets something).

I suggest something like `setUserInputSettingsFromCache`.